### PR TITLE
Added stylized versions of existing aliases

### PR DESCRIPTION
**Pull request by Jokero**
Closes https://github.com/dolsem/eslint-plugin-filename-rules/issues/1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,14 +23,14 @@ module.exports = {
     'filename-rules',
   ],
   rules: {
-    'filename-rules/match': [2, 'camelcase'],
+    'filename-rules/match': [2, 'camelCase'],
```
Comment (dolsem) [inline README.md:26]: Let's revert this example to the default style.
```suggestion
    'filename-rules/match': [2, 'camelcase'],
```

**Review by dolsem — CHANGES_REQUESTED** [review]
Otherwise, looks good, thank you!

File: README.md (hunk 1 above)
Comment (Jokero) [reply README.md:26]: What about the second example in readme?
```
'filename-rules/match': [2, { '.js': 'camelCase', '.ts': /^([a-z]+-)*[a-z]+(?:\..*)?$/ }]
```
I would also revert it then 😄

Comment (Jokero) [reply README.md]: Pushed changes

Comment (dolsem) [reply README.md]: Sorry for not being very clear - my idea was to keep the second as it is, so that we have an example of both in readme.

Comment (Jokero) [reply README.md]: No problem) I had the same thought. Updated

**Review by dolsem — APPROVED** [review]

**Pull request merged**